strings/is_palindrome.py

```diff
@@ -31,3 +31,36 @@ def is_palindrome(s):
             return False
         i, j = i+1, j-1
     return True
+
+
+"""
+Here is a bunch of other variations of is_palindrome function.
+
+Variation 1:
+Find the reverse of the string and compare it with the original string
+
+Variation 2:
+Loop from the start to length/2 and check the first character and last character
+and so on... for instance s[0] compared with s[n-1], s[1] == s[n-2]...
+
+"""    
+
+# Variation 1
+def string_reverse(s):
+	return s[::-1]
+
+def is_palidrome_reverse(s):
+	reversed_string = string_reverse(s)
+
+	if (s == reversed_string):
```

Comment:
you can write `return s == s[::-1]` in one line here.

Reply:
Yeah I thought of that but that would be awfully short. Also, I am working some new algorithms that are extremely short so I made this one a little longer than necessary. 😁